packages/core/src/extensions/lens-extension.ts

```diff
@@ -62,7 +63,7 @@ export class LensExtension<
 
   constructor({ id, manifest, manifestPath, isBundled }: InstalledExtension) {
     makeObservable(this);
-    this.id = id;
+    this.id = manifest.storeName || id;
```

Comment:
This doesn't seem right. Currently the id is absolute path to the package.json file.. which is probably arbitrary but it would seem inconsistent if some extensions suddenly had storeName as the id.